geemap/core.py

```diff
@@ -402,13 +402,25 @@ class Map(ipyleaflet.Map, MapInterface):
         "scroll_wheel_zoom": True,
     }
 
-    _BASEMAP_ALIASES: Dict[str, str] = {
-        "DEFAULT": "OpenStreetMap.Mapnik",
-        "ROADMAP": "Esri.WorldStreetMap",
-        "SATELLITE": "Esri.WorldImagery",
-        "TERRAIN": "Esri.WorldTopoMap",
-        "HYBRID": "Esri.WorldImagery",
-    }
+    if common.google_maps_api_key() is not None:
+        _BASEMAP_ALIASES: Dict[str, str] = {
+            "OpenStreetMap": "OpenStreetMap.Mapnik",
+            "Google.Roadmap": "Google.Roadmap",
+            "Google.Satellite": "Google.Satellite",
+            "Google.Terrain": "Google.Terrain",
+            "Google.Hybrid": "Google.Hybrid",
+            "Google.Traffic": "Google.Traffic",
+            "Google.Streetview": "Google.Streetview",
```

Comment:
Let's remove the traffic and street view layers. It's easier to add new functionality later rather than remove it and break users' scripts.

Reply:
Google Traffice and Streetview basemaps have been removed.